src/redux/nationSlice/NationSlice.jsx

```diff
@@ -0,0 +1,87 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const API_NOBEL = 'https://api.nobelprize.org/2.1/laureates';
+
+const initialState = {
+  nobels: [],
+  isLoading: true,
+  amount: 0,
+  uniqueCountries: [],
+};
+
+export const fetchNobel = createAsyncThunk(
+  'nobelSlice/fetchNobel',
+  async (name, thunkAPI) => {
+    try {
+      const res = await axios.get(API_NOBEL);
+      return res.data;
+    } catch (err) {
+      return thunkAPI.rejectWithValue('st went wrong');
+    }
+  },
+);
+
+export const takeUniqueCountries = (arr) => {
+  const uniqueCountries = [];
+  const compare = [];
+  // console.log('takeUniqueCountries'.toUpperCase());
```

Comment:
- [ ] Just as I suggested above, It is against the best practices to leave unused code as a comment on your code base. I'll suggest you remove this as it makes the codebase look a bit unprofessional. :+1: :smile: :wave: